### PR TITLE
codec(ticdc): simple protocol avro schema adjust to follow official convention.

### LINKED ISSUE #10455: simple protocol avro schema adjustment
Adjust the simple protocol avro schema, to make it follow the official convention.

### PR DESCRIPTION
**Pull request by 3AceShowHand**
### What problem does this PR solve?

Issue Number: close #10455 

### What is changed and how it works?

* Remove the `ID` from the `columnSchema`, since it's useless.
* Adjust avro schema, make it follow the official convention.

#### Tests 

 - Unit test
 - Integration test

### Release note 

```release-note
`None`
```

### CODE REVIEW

File: pkg/sink/codec/simple/message.json [hunk 1]
```diff
@@ -28,8 +28,10 @@
       },
       {
         "name": "elements",
-        "type": "array",
-        "items": "string"
+        "type": {
```
Comment (CharlesCheung96) [inline pkg/sink/codec/simple/message.json:31]: Please describe the reasons for such changes in the pr body.

Comment (3AceShowHand) [reply pkg/sink/codec/simple/message.json:31]: It's changed to follow the official convention.

**Review by CharlesCheung96 — APPROVED** [review]

**Pull request merged**